Source/WebCore/page/scrolling/ScrollAnchoringController.cpp

```diff
@@ -150,7 +163,8 @@ bool ScrollAnchoringController::didFindPriorityCandidate(Document& document)
 
         while (iterElement && iterElement.get() != elementForScrollableArea(m_owningScrollableArea)) {
             auto candidateResult = examineAnchorCandidate(*iterElement);
-            if (candidateResult == CandidateExaminationResult::Exclude || (iterElement == candidateElement && candidateResult == CandidateExaminationResult::Skip))
+            auto scrollAnchoringController = scrollAnchoringControllerForElement(*iterElement);
```

Comment:
I'd call this `elementsController` or something.

Reply:
Fixed.